src/main/java/com/rakovets/course/java/core/practice/decision_making_statements/Task01.java

```diff
@@ -38,6 +38,11 @@ public static void main(String[] args) {
     static boolean isTopPlayer(int sizeTop, int positionPlayerByDamage) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return false;
+        boolean topPlayer;
+        if (positionPlayerByDamage <= sizeTop) {
+            topPlayer = true;}
+        else {
+            topPlayer = false;}
+        return topPlayer;
```

Comment:
"Violation: There isn't separating white spaces (OR line break) for curly brake
Solution: Add white spaces"